controllers/object_controls.go

```diff
@@ -1309,6 +1310,7 @@ func transformForRuntime(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec,
 		sourceSocketFileName := path.Base(runtimeSocketFile)
 		// set envvar for runtime socket
 		var socketEnvvarName string
+		setContainerEnv(mainContainer, "RUNTIME_SOCKET", DefaultRuntimeSocketTargetDir+sourceSocketFileName)
```

Comment:
minor nit: can we move this so both `setContainerEnv()` calls are made in consecutive lines? E.g. move this to line 1319

Reply:
Sure

Reply:
Done, please check now